app/components/homepage/Mission.js

```diff
@@ -0,0 +1,29 @@
+import UtilStyles from "../../styles/utils.module.css";
+
+export default function Mission() {
+  return (
+    <div>
+      <section className={UtilStyles.missionSentence}>
+        <h1>Missao</h1>
+        <h2>Cada projeto é o início de uma nova história!</h2>
+        <p>
+          O Chalet - Design de Interiores nasce com a finalidade de trazer
+          ideias e soluções aos espaços dos seus clientes, quer sejam eles
+          residenciais ou comerciais, que combinem três premissas essenciais - o
+          conforto, a estética e a funcionalidade. Acreditamos que conseguimos
+          tornar realidade o espaço que o cliente idealizou, tendo sempre em
+          conta a sua identidade e objetivos adequados à sua vida e à da sua
+          família.
+        </p>
+      </section>
+      {/* {allPostsData.map(({ id, date, title }) => (
```

Comment:
please refrain from committing commented code

Reply:
okay, I will delete this code